lisa/executable.py

```diff
@@ -490,20 +496,20 @@ def __getitem__(self, tool_type: Union[Type[T], CustomScriptBuilder, str]) -> T:
             if not tool.exists:
                 tool_log.debug(f"'{tool.name}' not installed")
                 if tool.can_install:
-                    tool_log.debug(f"{tool.name} is installing")
+                    tool_log.debug(f"{tool.name} is about to be installed")
```

Comment:
I think `installing` is more useful. The logic between the log and installing, they are very closed. `installing` is a more closed state.

Reply:
No hard feelings for this one and I'll drop. But a native speaker shout here might help settle :)